src/modules/class-wpml-elementor-module-with-items.php

```diff
@@ -95,7 +95,7 @@ public function update( $node_id, $element, WPML_PB_String $string ) {
 	 * @return string
 	 */
 	private function get_string_name( $node_id, $value, $type, $key = '' ) {
-		return md5( $value ) . '-' . $type . $key . '-' . $node_id;
+		return $key . '-' . $type . '-' . $node_id;
```

Comment:
@BruceOnTheGo @brucepearson I don't think it can be a problem, because this method is only used in `\WPML_Elementor_Module_With_Items::update`. It will simply generate a new hash for the string name and put the translation on it.

Reply:
@adrianosferreira It's also used in the get function - \WPML_Elementor_Module_With_Items::get.

Please run some tests to make sure that this is going to work properly with text that was translated before this change.

Reply:
@andreasneuber hmm you found a bug. Let me double check it.

Reply:
@andreasneuber @BruceOnTheGo @brucepearson I slightly changed the way we build the name of the string for modules which contains Items. I included the Item ID in the end of the name:

```
return $key . '-' . $type . '-' . $node_id . '-' . $item_id;
```

@andreasneuber could you double check it?